index.html

```diff
@@ -0,0 +1,1091 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>New York Times</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <nav class="first-nav">
+    <div class="menu-icon">
+      <img src="images/menu.png" alt="Menu">
+    </div>
```

Comment:
This Hamburger icon should be a button. You can do this by using the button tag

Reply:
okay thanks for the review I did change the IMG with a button I hope it's good